tests/baselines/reference/treatWarningsAsErrorsFalse.errors.txt

```diff
@@ -0,0 +1,39 @@
+tests/cases/compiler/treatWarningsAsErrorsFalse.ts(1,14): warning TS6133: 'p' is declared but its value is never read.
+tests/cases/compiler/treatWarningsAsErrorsFalse.ts(2,5): warning TS7028: Unused label.
+tests/cases/compiler/treatWarningsAsErrorsFalse.ts(5,11): warning TS6133: 'y' is declared but its value is never read.
+tests/cases/compiler/treatWarningsAsErrorsFalse.ts(7,9): warning TS7029: Fallthrough case in switch.
+tests/cases/compiler/treatWarningsAsErrorsFalse.ts(9,14): error TS2678: Type '"b"' is not comparable to type '"a"'.
+tests/cases/compiler/treatWarningsAsErrorsFalse.ts(10,13): warning TS7030: Not all code paths return a value.
+tests/cases/compiler/treatWarningsAsErrorsFalse.ts(11,13): warning TS7027: Unreachable code detected.
+
+
+==== tests/cases/compiler/treatWarningsAsErrorsFalse.ts (7 errors) ====
+    function foo(p: any): string { // unused parameter
+                 ~
+!!! warning TS6133: 'p' is declared but its value is never read.
+        foo: while (false) {} // unused label
+        ~~~
+!!! warning TS7028: Unused label.
+    
+        const x: { kind: "a" } | { kind: "b" } = { kind: "a" };
+        const y = "any"; // unused variable
+              ~
+!!! warning TS6133: 'y' is declared but its value is never read.
+        switch (x.kind) {
+            case "a":
+            ~~~~
+!!! warning TS7029: Fallthrough case in switch.
+                void x; // implicit fallthrough
+            case "b":
+                 ~~~
+!!! error TS2678: Type '"b"' is not comparable to type '"a"'.

```

Comment:
This isn't strictly related to the test, but when I went to remove it, I considered that it is useful in verifying that we're not reporting _all_ errors as warnings.